x-pack/plugins/synthetics/public/apps/synthetics/components/test_run_details/components/console_icon.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as React from 'react';
+import {
+  EuiButtonIcon,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCodeBlock,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useRef, useState } from 'react';
+
+export const ConsoleInfoIcon = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const timer = useRef<NodeJS.Timeout | null>(null);
+
+  const closePopover = () => {
+    setIsPopoverOpen(false);
+  };
+
+  return (
+    <EuiPopover
+      button={
+        <div
+          onMouseEnter={() => {
+            // show popover with delay
+            if (timer.current) {
+              clearTimeout(timer.current);
+            }
+            timer.current = setTimeout(() => {
+              setIsPopoverOpen(true);
+            }, 300);
+          }}
+          onMouseLeave={() => {
+            if (isPopoverOpen) {
+              return;
+            } else if (timer.current) {
+              clearTimeout(timer.current);
+            }
+          }}
+          onClick={() => {
+            setIsPopoverOpen(!isPopoverOpen);
+          }}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter') {
```

Comment:
This needs some revision, if I tab my way to the icon button and press enter, the popover doesn't stay open as intended.

![20230815162900](https://github.com/elastic/kibana/assets/18429259/4ce55bab-71d8-492c-8343-bebb31e7fad9)